### PR TITLE
Announcement for Bazel 0.15

### PR DESCRIPTION
**Pull request by laurentlb**

### CODE REVIEW

File: _posts/2018-06-01-bazel-0.15.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: posts
+title: "Bazel 0.15"
+authors:
+  - laurentlb
+---
+
+The Bazel team is happy to announce a new version of Bazel, [Bazel 0.15](https://github.com/bazelbuild/bazel/releases/tag/0.15.0)!
+
+We'll be organizing the 2018 Bazel User Conference in NYC, on October 9-10. If you would like to join us, please register: https://goo.gl/forms/1ySgfiKXzntp6Knj2. Check out last year's session recordings on the [BazelCon website](https://sites.google.com/corp/bazel.build/conference2017/bazelcon2017/session-videos).
```
Comment (buchgr) [inline _posts/2018-06-01-bazel-0.15.md]: On a related note, we'll be ...

Comment (buchgr) [inline _posts/2018-06-01-bazel-0.15.md]: please register **at**:

Comment (laurentlb) [reply _posts/2018-06-01-bazel-0.15.md]: done

Comment (laurentlb) [reply _posts/2018-06-01-bazel-0.15.md]: done

**Pull request merged**